### PR TITLE
Update comments, formatting, and some minor code fixes

### PR DESCRIPTION
**Pull request by Goff-Davis**
Most of this is just formatting changes, but I added a couple comments and changed the wording on others. I also took out the non functional search. There are also some minor text changes.

### CODE REVIEW

File: frontend/src/components/creator-modal/creator-edit/creator-edit.tsx [hunk 1]
```diff
@@ -187,7 +188,9 @@ const CreatorEdit = (): ReactElement => {
 
         <div className="options-grading">
           <div className="participation">
-            <label htmlFor="participation-input">Participation Points:</label>
+            <label htmlFor="participation-input">
+              <span className="red">*</span> Participation Points:
```
Comment (weining-li) [inline frontend/src/components/creator-modal/creator-edit/creator-edit.tsx:192]: should probably also add `required` to the corresponding input

**Review by weining-li — APPROVED** [review]
:cake:

**Review by weining-li — APPROVED** [review]
¯\\\_(ツ)_/¯

File: frontend/src/components/creator-modal/creator-edit/creator-edit.tsx [hunk 2]
```diff
@@ -187,7 +189,9 @@ const CreatorEdit = (): ReactElement => {
...
```
Comment (weining-li) [inline frontend/src/components/creator-modal/creator-edit/creator-edit.tsx:193]: oh wait, same thing with `required` here as well ?

**Review by weining-li — COMMENTED** [review]
:O

File: frontend/src/components/creator-modal/creator-edit/creator-edit.tsx [hunk 3]
```diff
@@ -200,7 +204,9 @@ const CreatorEdit = (): ReactElement => {
           </div>
 
           <div className="correctness">
-            <label htmlFor="correctness-input">Correctness Points:</label>
+            <label htmlFor="correctness-input">
+              <span className="red">*</span> Correctness Points:
```
Comment (weining-li) [inline frontend/src/components/creator-modal/creator-edit/creator-edit.tsx:208]: i meant here smh

**Comment by weining-li** [comment]
why can't i delete my comments smh does github like people to clutter up the PR with comments :thinking: :thinking: :thinking:

**Comment by Goff-Davis** [comment]
Maybe because they're reviews and not comments ¯\_(ツ)_/¯

**Review by weining-li — DISMISSED** [review]

**Pull request merged**